doc/release/RELEASE-PROTOCOL.md

```diff
@@ -33,7 +33,7 @@ This document outlines the SDLC pillars of the opensupplyhub monorepo, as well a
 | v1.10.0 | March 23, 2024  | March 18, 2024 | @Nessa Drew |
 | v1.11.0 | April 20, 2024  | April 15, 2024 | @Nessa Drew |
 | v1.12.0 | May 18, 2024  | May 13, 2024 | @Vadim Kovalenko |
```

Comment:
Change code freeze date to `May 14, 2024 `

Reply:
I don't think we need to change that date in the protocol. I believe the postponement of the code freeze can simply be discussed with the team.

Reply:
> Change code freeze date to May 14, 2024 
I agree

Reply:
Okay.
Done.